docs/components/table.md

```diff
@@ -326,6 +327,86 @@ Example headers array:
 </script>
 ```
 
+#### Note about the `sortKey` option
+
+There may be times when data returned in a fetcher is not human-readable, but still needs to be displayed and  sorted in a table in a human-readble format. We cannot depend on the human-readable format sorting the column correctly - in these cases it is useful to specify which key in the data should be used to sort the column. An example of this would be when a date/time is returned in milliseconds.
```

Comment:
```suggestion
There may be times when data returned in a fetcher is not human-readable, but still needs to be displayed and sorted in a table in a human-readble format. We cannot depend on the human-readable format sorting the column correctly - in these cases it is useful to specify which key in the data should be used to sort the column. An example of this would be when a date/time is returned in milliseconds.
```